src/main/java/com/techsophy/tsf/runtime/form/service/impl/FormDataServiceImpl.java

```diff
@@ -88,25 +88,18 @@ public FormDataResponse saveFormData(FormDataSchema formDataSchema) throws IOExc
         Integer version = null;
```

Comment:
Need to understand what this code is doing, please explain, it looks unnecessarily complex